system/lib/symbols/wasm/libc++abi-mt-noexcept.symbols

```diff
@@ -450,7 +450,6 @@
 -------- W _ZnwmSt11align_val_tRKSt9nothrow_t
 -------- T __cxa_bad_cast
 -------- T __cxa_bad_typeid
--------- T __cxa_can_catch
```

Comment:
[These functions](https://github.com/emscripten-core/emscripten/blob/af5dbd4acdf965eff51e0d3111a1539344e3e7b5/system/lib/libcxxabi/src/private_typeinfo.cpp#L1298-L1319) are Emscripten-added functions that do not exist in the original libc++abi, and has been guarded with `__EMSCRIPTEN__`. The calls to these function are in [library_exceptions.js](https://github.com/emscripten-core/emscripten/blob/master/src/library_exceptions.js), which are used only when Emscripten EH is used. I changed `__EMSCRIPTEN__` to `__USING_EMSCRIPTEN_EXCEPTIONS__` in the ifdef guard, so these are now compiled only when Emscripten EH is used.

Reply:
Oh, looking at the CI errors, it seems they are used outside of Emscritpen EH.. will fix tomorrow.

Reply:
Did you find that they are still used? I would expect that they shouldn't be in wasm EH mode.

Reply:
I think they are gone for the right reason (I fixed the CI too). Before those functions were guarded under `#ifdef __EMSCRIPTEN__`, but I changed it to `#ifdef __USING_EMSCRIPTEN_EXCEPTION__`, because these functions are only used for Emscripten EH. So libraries with `-noexcept` build don't need these anymore. I also deleted these function names from some metadce test's exports/funcs file, because that test does not use EH.